### PR TITLE
Upstream SR2019 tasks

### PR DESCRIPTION
**Pull request by RealOrangeOne**
This PR upstreams some of the tasks created for SR2019, but never made their way into this repo.

See https://docs.google.com/document/d/1awPIduOEocfnioNT3A8OB8nN0-ggKA3Jered0bKiXrQ/edit?usp=sharing for some of the tasks we found to upstream.

### CODE REVIEW

File: comp/game/robot-flag-mounts.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+summary: Prepare robot flags for the competition
```
Comment (PeterJCLaw) [inline comp/game/robot-flag-mounts.yaml:1]: This title doesn't seem to match the filename, it also seems to just be a duplicate of `comp/game/robot-flags.yaml`. Should this be something else?

File: comp/livestream/configure-scenes.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Competition Prep
+
+description: >-
+  The livestream requires a number of different scenes and views, made up of
+  static images, video feeds, overlays etc.
```
Comment (PeterJCLaw) [inline comp/livestream/configure-scenes.yaml:11]: Is this just planning what the scenes will be, or also preparing the assets to use for those which are static? (Or do we already have something for the latter?)

File: comp/livestream/plan-video-feeds.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Competition Prep
+
+description: >-
+  The livestream will need to have a few different camera feeds to show different parts of the venue
+
+  - Arena (top down ideally, maybe more)
```
Comment (PeterJCLaw) [inline comp/livestream/plan-video-feeds.yaml:12]: This sounds like we should opt for top-down if there's only one, which I'm not sure is ideal? Suggest just leaving this without details.

File: comp/livestream/training.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+priority: major
+
+component: Competition
+
+milestone: $SRYYYY Competition
+
+description: >-
+  Volunteers who are assigned to run the livestream should be
+  trained how the system works, and how the event will be run
```
Comment (PeterJCLaw) [inline comp/livestream/training.yaml:11]:
```suggestion
  trained how the system works, and how the event will be run.
```

File: comp/livestream/configure-scenes.yaml (hunk 2 above)
Comment (RealOrangeOne) [reply comp/livestream/configure-scenes.yaml:11]: We already have the majority of the assets, but we should add a task to make sure we either check them, update them, or make new ones.

File: comp/pits/helpdesk/expert-list.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+milestone: $SRYYYY Competition
+
+description: >-
+  If there is a problem at helpdesk, that those on helpdesk cannot solve, they need to know who to escalate to.
+
+  There will be different experts depending on the domain.
+
+  Example domains:
+
+  - Rules
```
Comment (PeterJCLaw) [inline comp/pits/helpdesk/expert-list.yaml:16]:
```suggestion
  - Rules (probably whoever the head judge is)
```

File: comp/pits/helpdesk/print-documents.yaml [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+milestone: $SRYYYY Competition
+
+description: >-
+  Helpdesk will need a number of documents
+
+  - [ ] Rulebook
+  - [ ] Helpdesk instructions
+  - [ ] Kit swap form
+  - [ ] Venue map
```
Comment (PeterJCLaw) [inline comp/pits/helpdesk/print-documents.yaml:15]: Do we want to clarify that this may not be an exhaustive list?

**Review by PeterJCLaw — COMMENTED** [review]
Awesome, thanks for taking this on. I've commented a bunch of things inline, but mostly this looks great.

File: comp/livestream/configure-scenes.yaml (hunk 2 above)
Comment (PeterJCLaw) [reply comp/livestream/configure-scenes.yaml:11]: Maybe worth clarifying that? Also a link to the existing assets would be great.

File: comp/pits/helpdesk/expert-list.yaml (hunk 5 above)
Comment (RealOrangeOne) [reply comp/pits/helpdesk/expert-list.yaml:16]: It won't _just_ be the head judge, and there may be a number of people who know things about different parts of the rules (eg regulations vs game design). Leaving this unspecified forces the person to think about who may be best.

File: comp/livestream/hardware.yaml [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+description: >-
+  This ticket relates to providing the actual machine that will be
+  used for the livestream.
+
+  Requirements:
+  - Video mixer PC (With dedicated GPU, capable of running OBS)
+  - 2x HDMI capture card
+  - 2x mixers
+  - 2x preview monitors
+  - 2x cameras (depending on feed plan)
```
Comment (PeterJCLaw) [inline comp/livestream/hardware.yaml:18]:
```suggestion
  - 2× HDMI capture card
  - 2× mixers
  - 2× preview monitors
  - 2× cameras (depending on feed plan)
```

File: comp/livestream/hardware.yaml [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+milestone: $SRYYYY Competition Prep
+
+description: >-
+  This ticket relates to providing the actual machine that will be
+  used for the livestream.
+
+  Requirements:
+  - Video mixer PC (With dedicated GPU, capable of running OBS)
+  - 2× HDMI capture card
+  - 2× mixers
```
Comment (WillB97) [inline comp/livestream/hardware.yaml:16]: Can't really think why this needs 2 but we can always change this later.

**Review by WillB97 — APPROVED** [review]

File: comp/livestream/hardware.yaml (hunk 8 above)
Comment (RealOrangeOne) [reply comp/livestream/hardware.yaml:16]: This was copied from the previous ticket. @Tyler-Ward may know more.

Comment (Tyler-Ward) [reply comp/livestream/hardware.yaml:16]: I'm not sure where two mixers have come from. may have been munged from the two streamdecks used a couple of years ago.

A single audio mixer to mix the voice for the livestream would be good, an alternative option would be to do the mixing in OBS which would require an audio interface for the livestream presenters and the match presenters.

**Review by Tyler-Ward — APPROVED** [review]
That looks good to me.

**Pull request merged**